hatasmota/discovery.py

```diff
@@ -486,6 +504,8 @@ def get_entities_for_platform(
         entities.extend(get_status_sensor_entities(discovery_msg))
     elif platform == "switch":
         entities.extend(get_switch_entities(discovery_msg))
+    elif platform == "camera":
+        entities.extend(get_camera_entities(discovery_msg))
```

Comment:
Please move this before cover to keep the alphabetical sorting

Reply:
Done.